extensions/shared/plan-upgrade-notification.js

```diff
@@ -15,9 +16,40 @@ if ( undefined !== typeof window && window.location ) {
 	const { query } = parseUrl( window.location.href, true );
 
 	if ( query.plan_upgraded ) {
-		dispatch( 'core/notices' ).createNotice(
-			'success',
-			__( 'Plan upgraded. You may use premium blocks now.', 'jetpack' )
-		);
+		const planUrl = `https://wordpress.com/plans/my-plan/${ window.location.host }`;
```

Comment:
This does work but can you switch this to `window.Jetpack_Editor_Initial_State.siteFragment` — that value is then coming from the backend, just for consistency.

Reply:
..and also exists for .com simple sites. :-)

Reply:
Sounds good. 👍